protocol/check_test.go

```diff
@@ -1,42 +1,31 @@
-package xmrmaker
+package protocol
 
 import (
 	"context"
-	"math/big"
 	"testing"
 
 	contracts "github.com/athanorlabs/atomic-swap/ethereum"
 	"github.com/athanorlabs/atomic-swap/tests"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	ethcommon "github.com/ethereum/go-ethereum/common"
-	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCheckContractCode(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-	b := NewMockBackend(ctrl)
-
 	ec, chainID := tests.NewEthClient(t)
 	ctx := context.Background()
 	pk := tests.GetMakerTestKey(t)
 
 	txOpts, err := bind.NewKeyedTransactorWithChainID(pk, chainID)
 	require.NoError(t, err)
 
-	_, tx, _, err := contracts.DeploySwapFactory(txOpts, ec)
+	_, tx, _, err := contracts.DeploySwapFactory(txOpts, ec, ethcommon.Address{})
```

Comment:
Can we deploy it with a couple different addresses and verify that the gap that we don't check matches the expected addresses?

Reply:
yes, added